.conda/meta.yaml

```diff
@@ -4,30 +4,35 @@
 # {% set pyproject = load_file_data('../pyproject.toml', from_recipe_dir=True) %}
 # {% set project = pyproject.get('project') %}
 # {% set urls = pyproject.get('project', {}).get('urls') %}
+{% set name = "pyrovision" %}
+
 package:
-  name: pyrovision
+  name: {{ name|lower }}
   version: "{{ environ.get('BUILD_VERSION') }}"
 
 source:
-  fn: pyrovision-{{ environ.get('BUILD_VERSION') }}.tar.gz
-  url: ../dist/pyrovision-{{ environ.get('BUILD_VERSION') }}.tar.gz
+  fn: {{ name }}-{{ environ.get('BUILD_VERSION') }}.tar.gz
+  url: ../dist/{{ name }}-{{ environ.get('BUILD_VERSION') }}.tar.gz
 
 build:
+  number: 0
   noarch: python
-  script: python setup.py install --single-version-externally-managed --record=record.txt
+  script: {{ PYTHON }} setup.py install --single-version-externally-managed --record=record.txt
 
 requirements:
   host:
-    - python>=3.6, <4.0
+    - python>=3.6,<4.0
+    - pip
     - setuptools
-
+    - wheel
   run:
-    - python>=3.6, <4.0
-    - pytorch>=1.11.0, <2.0.0
-    - torchvision>=0.12.0, <1.0.0
+    - python>=3.6,<4.0
+    #- huggingface-hub>=0.4.0
```

Comment:
I understand it's in the pyproject.toml, but would you mind explaining why you remove it from here?
Is it because you default to pip install since there is no matching conda build?

Reply:
`huggingface-hub` is available only in the `Meepio` channel https://anaconda.org/Meepio/repo so `conda-build` can't install that package from the `main` channel.

Reply:
I see! But can't we simply add "Meepio" as a source channel for the build?

I'm trying to understand the reasoning for future decisions: why go for the pypi build if we can go for a conda build? (perhaps because meepio isn't the official HF conda channel?)

Reply:
if you don't mind, before moving on with the PR, would you mind sharing a bit more info on this @skupr-anaconda ? :)

I'm trying to better understand the reasoning for future similar situations

Reply:
It's not a usual situation for me too :)
If you can install your package without `huggingface-hub` you can simply add to the test/commands a new line:
` - pip install huggingface-hub`
In that way, you will be confident that all works correctly because `huggingface-hub` will be in the test environment.

Reply:
`pyro-vision` will be shipped without `huggingface-hub`. 
I guess `pyro-vision` can be installed and run **with** and **without** `huggingface-hub`. 
You only test that `pyro-vision` is running correctly **with** it. 
After `conda install pyro-vision`  the end-user can install `huggingface-hub`, for example, by `pip install huggingface-hub`.

Reply:
Alright, I thought about only leaving the conda option if all deps can be installed that way, but I guess it helps already :+1: 
However, that does mean that someone installing pyrovision via conda will need to additionally install HF hub on their own (hence updating the installation instructions) if I understand this correctly?

Reply:
Nice about my late answer @skupr-anaconda :upside_down_face: 
Final question: do you see a way to make sure that users, if they go for conda, don't have to go through additional installation steps? (because I see that as a regression for user installation otherwise)

Reply:
Yep, you can do it now because `huggingface_hub` (with **underscore**) is available on the default channel https://anaconda.org/main/huggingface_hub

Reply:
Oh that's perfect, would you mind updating your PR to add the dependency back. We can easily merge after this :)

Reply:
Done!